std/json.d

```diff
@@ -651,13 +695,13 @@ Params:
     maxDepth = maximum depth of nesting allowed, -1 disables depth checking
     options = enable decoding string representations of NaN/Inf as float values
 */
-JSONValue parseJSON(T)(T json, int maxDepth = -1, JSONOptions options = JSONOptions.none)
+JSONValue parseJSON(T)(T json, int maxDepth = -1, JSONOptions options = JSONOptions.none) @safe
```

Comment:
Fixed, thanks.

It troubles me that I can't indicate that my template does nothing un-@safe if its arguments don't, but that's nothing I can fix here.